improver_tests/wxcode/wxcode/test_wxcode_decision_tree.py

```diff
@@ -37,6 +37,7 @@
 from improver.wxcode.wxcode_decision_tree_global import (
     START_NODE_GLOBAL, wxcode_decision_tree_global)
 from . import check_diagnostic_lists_consistency
+from improver.wxcode.weather_symbols import WeatherSymbols
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [first party import "from improver.wxcode.weather_symbols import WeatherSymbols" should be placed before "from . import check_diagnostic_lists_consistency"](https://app.codacy.com/manual/metoppv_tech/improver/pullRequest?prid=5334158)